### PR TITLE
Add color-thief@2.0.1 with git autoupdate.

### PR DESCRIPTION
**Pull request by Piicksarn**
@Amomo  , would you mind to have a look at the pr?
The PR is referenced to issue  #6012.
Thank you!
The information of the pr:
The git repo : https://github.com/lokesh/color-thief/tree/v2.0.1

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `138`, Star `3929`, Fork `488`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

File: ajax/libs/color-thief/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "type": "git",
+    "url": "https://github.com/lokesh/color-thief.git"
+  },
+  "licenses": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lokesh/color-thief.git",
+    "basePath": "dist",
+    "files": [
+      "color-thief.min.js"
```
Comment (Amomo) [inline ajax/libs/color-thief/package.json]: I think we could use `**/*` here.

**Comment by Piicksarn** [comment]
@Amomo, I have it rebased and need your help for checkout the pr~
Thank you! :smile:

**Comment by Amomo** [comment]
@PeterDaveHello I think  this PR is fine, could you take a look?
Thank you! :-D

File: ajax/libs/color-thief/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "picture",
+    "photo",
+    "canvas"
+  ],
+  "homepage": "http://lokeshdhakar.com/projects/color-thief/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lokesh/color-thief.git"
+  },
+  "licenses": "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/color-thief/package.json]: licesnes?

**Comment by Piicksarn** [comment]
@Amomo, would you please have a look at the pr again?
Thank you!

**Pull request merged**